recipes/source.rb

```diff
@@ -37,21 +37,27 @@
   end
 end
 
+erlang_version     = node['erlang']['source']['version']
+erlang_url         = node['erlang']['source']['url'] || "http://erlang.org/download/otp_src_#{erlang_version}.tar.gz"
```

Comment:
^ that is functionally identical to the previous code, only working.

ruby variables >> attributes.